odoo/tools/convert.py

```diff
@@ -487,7 +487,7 @@ def _tag_menuitem(self, rec, parent=None):
         for child in rec.iterchildren('menuitem'):
             self._tag_menuitem(child, parent=menu.id)
 
-    def _tag_record(self, rec):
+    def _tag_record(self, rec, **kwargs):
```

Comment:
Using keyword args has no added value, since it is called as `self._tag_record(child_rec, **{inverse_name: record.id})` below.  Simply add an optional dict `extra` or `extra_vals` (defaulting to `()` to allow being passed to `update`).